server.js

```diff
@@ -71,47 +74,77 @@ app.get('/get-treatment', (req, res) => {
 });
 
 app.get('/describe/get-treatments', (req, res) => {
-
   res.type('text').send(`
     GET
       /get-treatments
 
     QUERY PARAMS
-      key:
-        This is the key used in the getTreatments call.
-      bucketing-key:
-        (Optional) This is the bucketing key used in the getTreatments call.
-      split-names:
-        List of comman-separated splits you want to include in the getTreatments call.
-      attributes:
-        (Optional) This should be a json string of the attributes you want to include in the getTreatments call.
-
-    EXAMPLE
-      curl 'http://localhost:4444/get-treatments?key=my-customer-key&split-names=my-experiment,another-experiment'
+       keys:
+         This is the array of keys to be used in the getTreatments call. Each key should specify a "matchingKey" 
+         and a "trafficType". You can also specify a "bucketingKey".
+       attributes:
+         (Optional) This should be a json string of the attributes you want to include in the getTreatments call.
+
+     EXAMPLE
+       curl 'http://localhost:4444/get-treatments?keys=\[\{"matchingKey":"my-first-key","trafficType":"account"\},
+       \{"matchingKey":"my-second-key","bucketingKey":"my-bucketing-key","trafficType":"user"\}\]
+       &attributes=\{"attribute1":"one","attribute2":2,"attribute3":true\}' -H 'Authorization={SPLITIO_EXT_API_KEY}'
   `);
-
 });
 
+// Returns the list of split names of a given traffic type
+function filterSplitsByTT(splitViews, trafficType) {
+  return reduce(splitViews, (acc, view) => {
+    if (view.trafficType === trafficType) {
+      acc.push(view.name);
+    }
+    return acc;
+  }, []);
+}
+
 app.get('/get-treatments', (req, res) => {
   const state = req.query;
-  const key = utils.parseKey(state.key, state['bucketing-key']);
+  let keys = [];
+  try {
+    keys = reduce(JSON.parse(state.keys), (acc, e) => {
+      acc.push(e);
+      return acc;
+    }, []);
```

Comment:
You could use map here instead of reduce, since you are returning everything

Reply:
I'll just remove the loop as I'm not doing processing here.